index.js

```diff
@@ -56,43 +61,43 @@ module.exports = (function (){
 
   return {
     api: async () => {
-      return await apiContract.at(Chain.API());
+      return await apiContract.at('0x7c728214be9a0049e6a86f2137ec61030d0aa964');
     },
 
     assetExchange: async () => {
-      return await assetExchangeContract.at(Chain.AssetExchange());
+      return await assetExchangeContract.at('0x8914a9e5c5e234fdc3ce9dc155ec19f43947ab59');
     },
 
     assetGenerator: async () => {
-      return await assetGeneratorContract.at(Chain.AssetGenerator());
+      return await assetGeneratorContract.at('0xf16165f1046f1b3cdb37da25e835b986e696313a');
     },
 
-    assetManagerEscrow: async () => {
-      return await assetManagerEscrowContract.at(Chain.BrokerEscrow());
-    },
+    // assetManagerEscrow: async () => {
+    //   return await assetManagerEscrowContract.at(Chain.BrokerEscrow());
+    // },
```

Comment:
Either remove them or leave them. In this case, why we are removing this?

Reply:
because I didn`t find BrokerEscrow address in Chain.js